.github/workflows/actions/setup_dev_ubuntu/install_libraries

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+set -x
+
+# Install ghc and cabal
+ghcup install ghc 8.10.7
+ghcup set ghc 8.10.7
+
+mkdir -p src
+mkdir -p srv/bin
```

Comment:
Is `srv` like a local prefix? Why not use `~/.local`?

Reply:
I'm just used to [/srv](https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/srv.html) but this probably is more an indication of my age than a good practice here.

🤔 might make sense to put it in .local Let's do that and see what happens in the next 1h30 🤣